polaris-react/src/components/OptionList/OptionList.tsx

```diff
@@ -130,24 +130,30 @@ export function OptionList({
           });
 
         return (
-          <li key={title || `noTitle-${sectionIndex}`}>
+          <Box
+            key={title || `noTitle-${sectionIndex}`}
+            as="li"
+            paddingBlockStart={isFirstOption ? undefined : '2'}
+          >
             {titleMarkup}
-            <ul
-              className={styles.Options}
-              id={`${id}-${sectionIndex}`}
-              role={role}
-            >
-              {optionsMarkup}
-            </ul>
-          </li>
+            <Bleed marginBlockStart={title ? undefined : '05'} marginInline="0">
+              <Box
+                as="ul"
+                id={`${id}-${sectionIndex}`}
+                role={role as BoxProps['role']}
```

Comment:
Nice 💯

Reply:
I do agree the one with all the comments was better for visibility to know to remove it but I'm okay with either version!